packages/vite-plugin-cloudflare/src/node-js-compat.ts

```diff
@@ -97,12 +99,10 @@ export function resolveNodeAliases(source: string, workerConfig: WorkerConfig) {
 	const from = source.slice(CLOUDFLARE_VIRTUAL_PREFIX.length);
 	const alias = preset.alias[from];
 
-	if (alias) {
-		return {
-			id: alias,
-			external: preset.external.includes(alias),
-		};
+	if (alias && preset.external.includes(alias)) {
+		throw new Error(`Alias to external: ${source} -> ${alias}`);
```

Comment:
I don't know if this is really necessary.
I haven't seen this error been thrown yet.
We can remove it if it is valid use case.